wordpress-mobile/WordPress-iOS
# Delete Site upgrades

Fixes #4841; fixes #4951

Adds site content export triggered directly; adds interface for cancelling deletion-blocking active upgrades; refactors interface flow for consistency with Android.

To test:
In the Advanced section of Site Settings for a WPcom blog you admin,
2. Trigger 'Export Content' and verify a download link is delivered to the contact email
3. Trigger 'Delete Site' and verify that if premium upgrades are active you are directed to the web interface for managing purchases. (Ideally, cancel them and verify that inactive upgrades no longer block deletion, but only do so with a throwaway site, as deletion is not user-reversible.)
4. Verify that design is broadly consistent with latest Android implementation.

Needs review: @kwonye, @adambbecker, @mattmiklic
